### ISSUE TITLE
Build fixture for testing "test" 

### ISSUE THREAD

**Issue by cgewecke** [cgewecke/buidler-gas-reporter#5]
Since this plugin uses mocha to test the execution of a set of mocha tests, behavior is a little weird. Something needs to be done to isolate the 'inner' tests from the plugin tests. 

There's also an issue with the buidler-ethers-project tests and `tsc`, but probably a config issue or perhaps a missing dep. Needs investigation.

**Comment by alcuadrado** [comment]
Can you post more info about these issues so I can help?

**Comment by cgewecke** [comment]
@alcuadrado Yes I will investigate further tomorrow AM (GMT -7) and post my findings.

**Comment by cgewecke** [comment]
@alcuadrado For a reason that's probably obvious but which I cannot see, the Ethers example tests error or won't compile because `this.env` is undefined in the test context there. 

If you're interested in taking a look, feel free. I will revisit over the weekend in any case.

Ethers should work as expected in `0.1.2`. Output in #6.

**Comment by alcuadrado** [comment]
I think the issue is that you are not including the plugin's type extensions in your `tsconfig.json`. Adding this should make it work:

```ts
"files": [
    "./node_modules/@nomiclabs/buidler-ethers/src/type-extensions.d.ts"
  ]
```

**Comment by alcuadrado** [comment]
There's another error. Something is throwing an exception, but I don't know why, but they aren't showing in mocha's output.

**Comment by cgewecke** [comment]
@alcuadrado You've fixed this here with #8, closing. Only thing that remains is to unskip the ethers test when buidler beta.9 is published. Opening a separate ticket for that.

**Comment by alcuadrado** [comment]
Great! I'll create another PR for that when beta.9 gets released.
